src/routes/UserRoutes.ts

```diff
@@ -21,6 +21,10 @@ export class UserRoutes {
 
 		router.post('/authenticate', this.userController.authenticate.bind(this.userController));
 
+		router.post('/forgot_password', getUser(TokenType.Auth), this.userController.forgotPassword.bind(this.userController));
```

Comment:
```suggestion
		router.post('/forgot_password', this.userController.forgotPassword.bind(this.userController));
```

Users may not be able to login, so we can't require that they have a valid authorization token. The only thing we can do is take the user's email address.